src/connect4/core.rs

```diff
@@ -55,6 +57,17 @@ pub enum MyColor {
     Red,
 }
 
+impl MyColor {
+    pub fn get_draw_color(&self) -> ggez::graphics::Color {
+        let circ_color = match self {
+            MyColor::White => graphics::WHITE,
+            MyColor::Blue => graphics::Color::from_rgba(0,0,255,255),
+            MyColor::Red => graphics::Color::from_rgba(255,0,0,255),
+        };
+        circ_color
+    }
+}
+
```

Comment:
Added conversion between `MyColor` and `graphics::Color` cuz we need that in more than one place now